build.sh

```diff
@@ -15,54 +15,10 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
```

Comment:
Let's turn on the usual error handling
```suggestion

set -o errexit
set -o nounset
set -o pipefail
```